theseus_gui/src/components/ui/Navigation.vue

```diff
@@ -0,0 +1,155 @@
+<script setup>
+import { SearchIcon, BookIcon, ClientIcon, PlusIcon, SettingsIcon } from 'omorphia'
+import { useInstances } from '@/store/state'
+import { RouterLink, useRoute } from 'vue-router'
+import SideNav from '@/components/SideNav.vue'
+import UserSection from '@/components/ui/UserSection.vue'
+import Instance from '@/components/ui/Instance.vue'
+const instances = useInstances()
+
+instances.fetchInstances()
+const route = useRoute()
+</script>
+
+<template>
+  <SideNav class="navigation">
+    <UserSection />
+    <div class="pages-list">
+      <RouterLink
+        to="/"
+        class="omorphia__button button-base padding-block-sm padding-inline-lg radius-md standard-button"
+      >
+        <ClientIcon />Home</RouterLink
+      >
+      <RouterLink
+        to="/browse"
+        class="omorphia__button button-base padding-block-sm padding-inline-lg radius-md standard-button"
+      >
+        <SearchIcon />Browse</RouterLink
+      >
+      <RouterLink
+        to="/library"
+        class="omorphia__button button-base padding-block-sm padding-inline-lg radius-md standard-button"
+      >
+        <BookIcon />Library</RouterLink
+      >
+    </div>
+    <div class="instance-list">
+      <p>Instances</p>
+      <Instance v-for="instance in instances.instances" display="list" :instance="instance" />
+    </div>
+    <RouterLink
+      to="/add-instance"
+      class="omorphia__button button-base padding-block-sm padding-inline-lg radius-md standard-button"
+      id="add-instance-btn"
+    >
+      <PlusIcon />
+      Create Instance
+    </RouterLink>
+
+    <div id="settings">
+      <RouterLink
+        to="/settings"
+        class="omorphia__button button-base padding-block-sm padding-inline-lg radius-md standard-button"
+        ><SettingsIcon /> Settings</RouterLink
```

Comment:
button shouldn't be center aligned and the settings icon should have more margin between the text